lib/defaults/defaults.go

```diff
@@ -28,13 +28,15 @@ import (
 	"strings"
 	"time"
 
+	"github.com/coreos/go-semver/semver"
 	"github.com/gravitational/trace"
 	"github.com/jonboulle/clockwork"
 	"golang.org/x/crypto/ssh"
 
 	"github.com/gravitational/teleport"
 	"github.com/gravitational/teleport/api/defaults"
 	"github.com/gravitational/teleport/lib/limiter"
+	"github.com/gravitational/teleport/lib/modules"
```

Comment:
`lib/modules` pulls in a few other libs, which i think we should avoid for defaults lib (also those libs can easily run into circular dependency issues).

any easy way to avoid it? interface? or maybe move it to modules?

Reply:
moved to a new package `teleportassets`: https://github.com/gravitational/teleport/compare/64f6d4cb401b36a6ef707e6fe9a1180d12569054..dda8cc7c4885e9978a3a3b700ed9f19b8908cffa